app/javascript/Pages/Home.jsx

```diff
@@ -1,30 +1,38 @@
-import React from "react";
+import React, { Component } from "react";
 import Layout from "./Layout";
+import Navbar from "./components/Navbar/Navbar";
+import Footer from "./components/Footer/Footer";
 import { Head, Link } from "@inertiajs/inertia-react";
+import { Inertia } from "@inertiajs/inertia";
 import "../stylesheets/body.css";
+import "../stylesheets/Home.css";
 
-export default function Welcome(props) {
-  const token = document.querySelector(
-    "[name=csrf-token]"
-  ).content;
-  const headers = { "X-CSRF-Token": token };
+class Homepage extends Component {
+  render() {
+    const token = document.querySelector("[name=csrf-token]").content;
```

Comment:
**suggestion (non-blocking)**: same for this token, if you need it in multiple pages you should move this to the entry file and either use a store or context to pass it to the components